test/junit/JUnitTests.gmk

```diff
@@ -41,7 +41,6 @@ $(BUILDTESTDIR)/JUnitTests.ok: \
 	$(RM) $(@:%.ok=%/work) $(@:%.ok=%/report)
 	$(MKDIR) -p $(@:%.ok=%/work) $(@:%.ok=%/report)
 	cd $(@:%.ok=%/work) ; \
-	set -o pipefail ; \
```

Comment:
Hello Christian, is this an intentional change on this line?

Reply:
Yes. That `set` leads to:
```
...
/bin/sh: 2: set: Illegal option -o pipefail
make: *** [../test/junit/JUnitTests.gmk:43: /.../apidiff/build/test/JUnitTests.ok] Error 2
Error: Process completed with exit code 2.
```
in https://github.com/sormuras/apidiff/actions/runs/11271903419/job/31345716315#step:5:61